specification/columns/effectivecost.md

```diff
@@ -7,7 +7,11 @@ This column resolves two challenges that are faced by practitioners:
 1. Practitioners need to *amortize* relevant purchases, such as upfront fees, throughout the *commitment* and distribute them to the appropriate reporting groups (e.g. [*tags*](#glossary:tag), [*resources*](#glossary:resource)).
 2. Many [*commitment discount*](#glossary:commitment-discount) constructs include a recurring expense for the *commitment* for every [*billing period*](#glossary:billing-period) and must distribute this cost to the *resources* using the *commitment*. This forces reconciliation between the initial *commitment* [*row*](#glossary:row) per period and the actual usage *rows*.
 
-The EffectiveCost column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. EffectiveCost MUST be 0 when ChargeCategory is "Purchase" and the purchase is intended to cover future eligible charges. The aggregated EffectiveCost for a billing period may not match the charge received on the invoice for the same *billing period*.
+The EffectiveCost column adheres to the following requirements:
+
+* The EffectiveCost column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null.
+* This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency.
+* EffectiveCost MUST be 0 when ChargeCategory is "Purchase" and the purchase is intended to cover future eligible charges. The aggregated EffectiveCost for a billing period may not match the charge received on the invoice for the same *billing period*.
```

Comment:
```suggestion
* EffectiveCost MUST be 0 when ChargeCategory is "Purchase" and the purchase is intended to cover future eligible charges. 
* The aggregated EffectiveCost for a billing period may not match the charge received on the invoice for the same *billing period*.
```

Reply:
I like this callout about the possibility of the aggregated EffectiveCost not matching the invoice total, but does it belong in the normative section?
Also, fingers crossed #615 will obviate the need for this comment.

Reply:
I'll copy this comment to the WorkItem #557. Similar callouts exist in some other columns - e.g., Billed Cost. 

> The sum of the BilledCost for rows in a given billing period MUST match the sum of the invoices received for that billing period for a billing account.

We can discuss and address these across the board as part of the WorkItem #557.

On that note, @kk09v,  I would appreciate your feedback on the [24.11.26 - Normative Requirements Refinement](https://docs.google.com/document/d/1fKeGCBIFQqx9puEpWdIT9CpGSn_-Gh86ky1bf_5-x5Y/edit?tab=t.0), particularly the Guidelines/Principles and the Template/Cookbook.